project_task_category/models/project_task_category.py

```diff
@@ -0,0 +1,15 @@
+# -*- coding: utf-8 -*-
+# © 2016 Elico Corp (www.elico-corp.com).
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import fields, models
+
+
+class ProjectTask(models.Model):
+    _inherit = 'project.task'
+
+    categ_id = fields.Many2one(
+        'project.category',
+        'Category',
+        required=True
```

Comment:
Why do we need the Category to be always mandatory?
I can see the case for it to be mandatory depending on the Project...
How about having it mandatory only on the Task Form? That way it will be easier to customize more specific use cases (and to have it installed on databases already having Task data).